tempto-core/src/main/java/io/prestosql/tempto/assertions/QueryAssert.java

```diff
@@ -448,7 +448,7 @@ public Row(Object... values)
 
         public Row(List<Object> values)
         {
-            this.values = requireNonNull(values, "values is null");
+            this.values = unmodifiableList(new ArrayList<>(requireNonNull(values, "values is null")));
```

Comment:
why not `ImmutableList`? I see we use both, it would be nice to decide and use one coherently.

Reply:
Because a row can contain null values.